moto/cognitoidp/models.py

```diff
@@ -439,10 +437,8 @@ def __init__(
         self.access_tokens: Dict[str, Tuple[str, str]] = {}
         self.id_tokens: Dict[str, Tuple[str, str]] = {}
 
-        with open(
-            os.path.join(os.path.dirname(__file__), "resources/jwks-private.json")
-        ) as f:
-            self.json_web_key = jwk.RSAKey.import_key(json.loads(f.read()))
+        jwks_file = load_resource(__name__, "resources/jwks-private.json")
```

Comment:
This is a JSON file, so this is now also compressed, and needs to be handled as such. 

Compressing isn't really necessary for this particular file, as it is very small, but it does make things consistent.